src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -976,8 +977,17 @@ def __call__(
 
         # 8. Denoising loop
         num_warmup_steps = len(timesteps) - num_inference_steps * self.scheduler.order
+        is_unet_compiled = is_compiled_module(self.unet)
+        is_controlnet_compiled = is_compiled_module(self.controlnet)
+        is_torch_higher_equal_than_2_1 = version.parse(version.parse(torch.__version__).base_version) >= version.parse(
```

Comment:
```suggestion
        is_torch_higher_equal_2_1 = version.parse(version.parse(torch.__version__).base_version) >= version.parse(
```

Reply:
Also we have utile for this that might be good using here: https://github.com/huggingface/diffusers/blob/8dba1808852e7f5c08f91296006ec254cecdd1b1/src/diffusers/utils/import_utils.py#L583